static/app/views/settings/organizationMembers/organizationMemberDetail.spec.tsx

```diff
@@ -160,16 +157,21 @@ describe('OrganizationMemberDetail', function () {
     });
 
     it('changes org role to owner', async function () {
-      render(<OrganizationMemberDetail params={{memberId: member.id}} />, {
-        context: routerContext,
-      });
+      const {routerContext, routerProps} = initializeOrg({organization});
```

Comment:
I think you can just inline the org overrides here

```tsx
initalizeOrg({
  organization: {teams, features: ['team-roles']}
});
```

Reply:
Do we actually need to call `initializeOrg` in every test?

Reply:
Ah yeah some tests change it, cool cool cool

Reply:
ya they do :( 

i started with some global consts but then eslint said no